LinkedList.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  LinkedList.swift
+//  Collections
+//
+//  Adapted and modified from Airspeed Velocity and Chris Eidhof. 
+//  Created by Jeremy Corren on 12/28/16.
+//
+//
+
+/// The `LinkedList`.
+public enum LinkedList<T> {
```

Comment:
I've gotten in the habit, particularly in terse declarations, of putting whitespace before the angle brackets, like this:

```Swift
enum LinkedList <T> { }
```

For some reason, it feels nice. So, maybe at least for consistency?